spec/lib/saml_replay_cache_spec.rb

```diff
@@ -12,24 +12,42 @@ def create_saml_response(
     session_not_on_or_after: now + 2.hours
   )
     response_xml = <<~XML
-    <?xml version="1.0"?>
-    <samlp:Response xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion" xmlns:ds="http://www.w3.org/2000/09/xmldsig#"
-                    ID="_response123"
-                    IssueInstant="#{now.iso8601}">
-      <saml:Assertion ID="#{assertion_id}"
-                      IssueInstant="#{now.iso8601}">
-        <saml:Issuer>https://idp.cat.com</saml:Issuer>
-        <ds:Signature>
-          <ds:SignedInfo>
-            <ds:Reference URI="##{assertion_id}"/>
-          </ds:SignedInfo>
+    <samlp:Response
```

Comment:
Where does all of this come from? Is it generated somewhere?

Reply:
See `/t/150000` internally.